doc/mgmt/Management Framework.md

```diff
@@ -1029,7 +1335,58 @@ These utilities methods provides below mentioned common operations on the YGOT s
         string - object field name of the given target object
         error - error object to describe the error if this methods fails to perform the desired operation, otherwise nil
 
-###### 3.2.2.6.4 DB access layer
+###### 3.2.2.6.4 YANG Model Versioning
+
+###### 3.2.2.6.4.1 Version Management
```

Comment:
which rest module does the version management ?

Reply:
Version management logic will be in translib -- it is common to both REST and gNMI services.